__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -341,7 +341,7 @@ export const ControlledMultilpleCombobox: Story = {
         await userEvent.keyboard("{Enter}");
 
         // Assert
-        expect(canvas.getByRole("log")).toHaveTextContent(
+        expect(canvas.getAllByRole("log")[0]).toHaveTextContent(
```

Comment:
Note: This started to fail with the Storybook global decorator, as there were multiple `role="log"` elements at that point!